src/ui/CMakeLists.txt

```diff
@@ -140,9 +136,9 @@ if(EMSCRIPTEN)
     app
     DEPENDS ${outputFileName})
 
-  # TODO should be pulled via linking od_acquisition, but that's outside the source folder (src/ui) for the ui-wasm
-  # build
+  # TODO should be pulled via linking the targets, but they're outside the source folder (src/ui) for the ui-wasm build
   target_include_directories(${target_name} PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/../acquisition) # for daq_api.hpp
+  target_include_directories(${target_name} PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/../utils/include)
```

Comment:
I wish we wouldn't need those hacks...
Maybe we can refactor the build later so that everything builds the top directory, and add flags to disable service or UI??